README.rst

```diff
@@ -63,17 +63,23 @@ templates, add this to your app's ``urls.py``:
     from crudlfap import crudlfap
     from .models import Server
 
-    urlpatterns = crudlfap.Router(Server, fields='__all__', menus=['main'],
-        allow=lambda view, user: return True, # Default requires is_staff !
+    urlpatterns = crudlfap.Router(
+        Server,
+        fields='__all__',
+        menus=['main'],
+        allow=lambda view,
+        user: return True, # Default requires is_staff !
```

Comment:
This is confusing cause it's not a dict, but looks like a dict key now. Also, I think the return must be ditched here, since it's implicit

Reply:
Ooops. I messed up the lambda bit... Too much haste.

Is that the part you object to or do you not like function calls split onto multiple lines like this? I do it all the time so you'd hate my codebase.